frontend/app/views/spree/products/_cart_form.html.erb

```diff
@@ -29,7 +29,7 @@
     <% end %>
 
     <% if @product.price_for(current_pricing_options) and !@product.price.nil? %>
-      <div data-hook="product_price" class="columns five <% if !@product.has_variants? %> alpha <% else %> omega <% end %>">
+      <div data-hook="product_price" class="columns five <%= @product.has_variants? ? 'alpha' : 'omega' %>">
```

Comment:
I think this is a fine change in principle, but the previous code had a `!` before the main condition, turning the two branches around. Maybe rather: 

```ruby
@product.has_variants? ? 'omega' : 'alpha'
```